libtransact/src/context/mod.rs

```diff
@@ -98,35 +97,42 @@ impl<K: PartialEq + Clone + Debug, V: Clone + Debug> Context<K, V> {
     }
 
     /// Adds StateChange::Set without deleting previous StateChanges associated with the Key
-    pub fn set_state(&mut self, key: K, value: V) {
-        let new_state_change = StateChange::Set { key, value };
+    pub fn set_state(&mut self, key: &str, value: &[u8]) {
```

Comment:
These should be owned values as the previous signature.

Reply:
Addressed